hls4ml/templates/vitis/nnet_utils/nnet_sepconv1d_stream.h

```diff
@@ -86,7 +86,7 @@ void separable_conv_1d_cl(hls::stream<data_T> &data, hls::stream<res_T> &res,
     #pragma HLS DATAFLOW
 
     hls::stream<dw_res_T> depthwise_res;
-    unsigned res_depth = CONFIG_T::depthwise_config::out_width;
+    const unsigned res_depth = CONFIG_T::depthwise_config::out_width;
```

Comment:
Does using `const` here bring any benefit? And since this is a newer compiler, why not `constexpr`?